ci/Jenkinsfiles/build.groovy

```diff
@@ -57,7 +57,7 @@ def isPullRequest() {
 }
 
 String getVersion() {
-  return isPullRequest() ? getPullRequestVersion() : getReleaseVersion()
+  return isPullRequest() ? readMavenPom().getVersion() : getReleaseVersion()
```

Comment:
Wouldn't it be a problem for docker images? I'm talking about collision on our Docker registry.

Reply:
You're right... I knew we missed something @ataillefer 😂